about.html

```diff
@@ -0,0 +1,168 @@
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus, minima saepe incidunt quis ex eum voluptatem rem
+      unde fuga repudiandae iure? Animi porro laboriosam odio dolore quisquam nam veritatis iste.<br><br> Lorem ipsum
+      dolor sit amet consectetur adipisicing elit. Non deleniti laboriosam quas, consequuntur sint hic adipisci
+      perferendis nisi corrupti odit dolores similique fugit minus libero optio quos rerum suscipit quae.<br><br>
+      Similique corporis in illo delectus! Commodi ducimus cum quam expedita, qui sit nisi itaque, asperiores officiis
+      facilis in maxime nesciunt voluptatibus. Quaerat debitis ea quidem? Minus, cupiditate? Soluta, eaque
+      dolorum.<br><br>
+      Ea quos labore officiis explicabo possimus? Suscipit beatae aliquid ipsam ab nisi perferendis aut magni! A quos
+      voluptate, cum eius, dicta quisquam sapiente dolor ea possimus, dignissimos quod delectus ipsa.<br><br>
+      Fugiat facere, dolorum cumque provident nisi dolore. Quo tempora suscipit accusantium perspiciatis quos aliquid
+      tenetur inventore ullam ut quae? Vel optio inventore dignissimos repellat unde magni beatae! Molestiae, optio ab?
+    </p>
+    <p class="event-description desktop-none">Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus, minima
+      saepe incidunt quis ex eum voluptatem rem unde fuga repudiandae iure? Animi porro laboriosam odio dolore quisquam
+      nam veritatis iste.<br><br> Lorem ipsum dolor sit amet consectetur adipisicing elit. Non deleniti laboriosam
+      quas, consequuntur sint hic adipisci perferendis nisi corrupti odit dolores similique fugit minus libero optio
+      quos rerum suscipit quae.<br><br></p>
+  </section>
+  <!-- Contact-->
+  <section>
+    <div class="contact-event">
+      <h3 class="contact-event_title">
+        For more information about the Energy event, please contact us:
+      </h3>
+      <a href="mailto:name@email.com" class="contact-event_mail">energy2024@events.com</a>
+    </div>
+  </section>
+  <section class="logo-info">
+    <h2>World Energy Conferance </h2>
+    <p>The World Energy Conferance has a new logo. Lorem ipsum dolor sit amet consectetur adipisicing elit. <br><br>Impedit doloremque officia aliquam debitis ab quasi nesciunt quaerat? Debitis voluptas corporis sapiente vero ab, minus possimus officia, nisi, deserunt nulla sint?</p>
+    <div class="logo-info_logo">
+      <img src="assets/images/advanced-energy-vector-logo.png" alt="logo info">
+    </div>
+  </section>
+  <!-- History -->
+  <section class="history">
+    <hr>
+    
+    <div>
+      <h2>Great past Energy Event</h2>
+    </div>
+    <p>
+      Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+    </p>
+    <section class="history-images">
+
+    <div class="history-cards" id="energy2019">
+      2019<br>Amsterdam
+    </div>
+    <div class="history-cards" id="energy2020">
+      2020<br>New York</div>
+      </section>
+      </section>
```

Comment:
**Suggested Changes**
- [ ] This section doesn't resemble the given design in terms of padding, and margin between elements. We can do better than this. Let us try to reduce the padding and get as close as  possible to the design. I have attached a screenshot of your design and that provided for comparison.

Your Design
![your-summit](https://user-images.githubusercontent.com/7220167/143538550-db7a1054-6069-4727-92e2-d4a858262eed.png)

Given Design
![summit](https://user-images.githubusercontent.com/7220167/143538581-eabd39b2-66b7-44c1-a977-891976bb33f2.png)

Reply:
Fixed